incubator/hnc/pkg/controllers/config_controller.go

```diff
@@ -42,13 +56,24 @@ func (r *ConfigReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	// achieve the reconciliation.
 	r.Log.Info("Reconciling cluster-wide HNC configuration.")
 
+	// TODO: This is only for debuging and testing in the early development stage. Remove this when the
+	// implementation is completed.
+	r.logSpec(inst)
+
```

Comment:
Maybe remove this now? And just put out logs whenever something interesting happens (which I think you're already doing).

Reply:
Sure! Done!